ngraph_onnx/onnx_importer/ops_bridge.py

```diff
@@ -679,7 +679,9 @@ def Split(onnx_node, ng_inputs):  # type: (NodeWrapper, List[NgraphNode]) -> Tup
     count_outputs = len(onnx_node.get_output_names())
     axis_to_split = onnx_node.get_attribute_value('axis', 0)
 
-    if axis_to_split < 0 or axis_to_split >= len(data.shape):
+    if axis_to_split < 0:
+        axis_to_split = len(data.shape) + axis_to_split
```

Comment:
Add a comment to explain what we're doing here.

Reply:
This line is required to cover python negative indexing because backend is written in c++ and it will crash. This code convert it to proper index.